### ISSUE TITLE
What is the difference?

### ISSUE THREAD

**Issue by grumas-rc** [semiotic-ai/autoagora-processor#2]
Hello!
In [README.md](https://github.com/semiotic-ai/autoagora-processor/blob/master/README.md), configuration section has parameters:
```
  --graph-postgres-host GRAPH_POSTGRES_HOST
                        URL of the postgres instance use by the graph-nodes. [env var: GRAPH_POSTGRES_HOST] (default:
                        None)
  --graph-postgres-database GRAPH_POSTGRES_DATABASE
                        Name of the graph-node database. [env var: GRAPH_POSTGRES_DATABASE] (default: None)
  --graph-postgres-username GRAPH_POSTGRES_USERNAME
                        Username for the graph-node databse. [env var: GRAPH_POSTGRES_USERNAME] (default: None)
  --graph-postgres-password GRAPH_POSTGRES_PASSWORD
                        Password for the graph-node database. [env var: GRAPH_POSTGRES_PASSWORD] (default: None)
```
```
  --postgres-host POSTGRES_HOST
                        URL of the postgres instance use by the graph-nodes. [env var: POSTGRES_HOST] (default: None)
  --postgres-database POSTGRES_DATABASE
                        Name of the graph-node database. [env var: POSTGRES_DATABASE] (default: None)
  --postgres-username POSTGRES_USERNAME
                        Username for the graph-node databse. [env var: POSTGRES_USERNAME] (default: None)
  --postgres-password POSTGRES_PASSWORD
                        Password for the graph-node database. [env var: POSTGRES_PASSWORD] (default: None)
```
Is there a mistake in the description, or is it really necessary to specify the graph-node database twice?

**Comment by haritowa** [comment]
I believe the second one is logs db

**Comment by aasseman** [comment]
@haritowa  is right.
The descriptions for the `--postgres-...` options are wrong indeed.

**Comment by aasseman** [comment]
Fixed in #5
